.github/workflows/python.yaml

```diff
@@ -15,8 +15,69 @@ jobs:
 #      if: ${{ github.event_name == 'pull_request' }}
 #      with:
 #        access_token: ${{ github.token }}
+  do_python:
+    if: |
+      github.event_name != 'pull_request' ||
+      (
+        !contains(github.event.pull_request.body, '[no Python]')
```

Comment:
I don't know if it is better or worse, but we can also trigger this on comments: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#discussion_comment

Reply:
We could. We would have to trigger all our workflows on every comment an then make sure the jobs don't duplicate too much work, or everything is skipped in most cases. I'm not sure if github would leave previous successes in place if a later workflow run skips the previously succeeding job.

This seems tricky, and I wanted to keep this basic.